app/index/template.hbs

```diff
@@ -0,0 +1,23 @@
+
+
+
+
+
+
+
+<div class="run-list">
+  {{#each model as |run|}}
+  {{#link-to "detail" model._id}}
```

Comment:
I know I just turned model into run but I thought I might still have to reference it by its original name.  just tried changing it to run, and the link takes me somewhere now.  I have a new error but it looks like it might be sass related.  I'll try to knock it out myself.  Thanks!